hydra-access-controls/spec/unit/visibility_spec.rb

```diff
@@ -11,6 +55,18 @@ class MockParent < ActiveFedora::Base
     include VisibilityOverride
   end
 
+  class Foo < ActiveFedora::Base
```

Comment:
Please move this to a before block and then `remove_const(:Foo)` in an after block.